clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_vector_rbac.yaml

```diff
@@ -0,0 +1,11 @@
+---
+# oc prjoect api-audit-log; oc adm policy add-scc-to-user privileged -z logging-app
```

Comment:
Can we put that into the yaml to keep this reproducible?

Reply:
Done.

Maybe `scc/priviledged` is over-permitted.
I tried with `scc/hostaccess` and did not work out.

```bash
  Warning  FailedCreate  5s (x12 over 15s)  daemonset-controller  Error creating: pods "vector-audit-log-" is forbidden: unable to validate against any security context constraint: [spec.volumes[0]: Invalid value: "hostPath": hostPath volumes are not allowed to be used spec.containers[0].securityContext.privileged: Invalid value: true: Privileged containers are not allowed spec.containers[0].securityContext.privileged: Invalid value: true: Privileged containers are not allowed]
```